modules/apigee_edge_teams/apigee_edge_teams.services.yml

```diff
@@ -130,3 +130,12 @@ services:
     arguments: ['@entity_type.manager', '@logger.channel.apigee_edge_teams']
     tags:
       - { name: paramconverter }
+
+  paramconverter.apigee_edge_teams.entity_uuid:
+    parent: paramconverter.entity
+    class: Drupal\apigee_edge_teams\ParamConverter\InvitationsEntityUuidConverter
```

Comment:
@audave 
Why do you need a paramconverter which is already there in JsonAPI module (EntityUuidConverter)? 
This will also create a dependency of JsonAPI module.
IMO you should make use of uuid() and EntityUuidConverter instead of a new converter.